core/src/components/textarea/textarea.tsx

```diff
@@ -165,6 +165,54 @@ export class Textarea implements ComponentInterface {
    */
   @Prop({ mutable: true }) value?: string | null = '';
 
+  /**
+   * If `true`, a character counter will display the ratio of characters used and the total character limit.
+   * Developers must also set the `maxlength` property for the counter to be calculated correctly.
+   */
+  @Prop() counter = false;
+
+  /**
+   * A callback used to format the counter text.
+   * By default the counter text is set to "itemLength / maxLength".
+   */
+  @Prop() counterFormatter?: (inputLength: number, maxLength: number) => string;
+
+  /**
+   * Text that is placed under the textarea and displayed when an error is detected.
+   */
+  @Prop() errorText?: string;
+
+  /**
+   * The fill for the item. If `'solid'` the item will have a background. If
+   * `'outline'` the item will be transparent with a border. Only available in `md` mode.
+   */
+  @Prop() fill?: 'outline' | 'solid';
+
+  /**
+   * Text that is placed under the textarea and displayed when no error is detected.
+   */
+  @Prop() helperText?: string;
+
+  /**
+   * The visible label associated with the textarea.
+   */
+  @Prop() label?: string;
+
+  /**
+   * Where to place the label relative to the input.
+   * `'start'`: The label will appear to the left of the input in LTR and to the right in RTL.
+   * `'end'`: The label will appear to the right of the input in LTR and to the left in RTL.
+   * `'floating'`: The label will appear smaller and above the input when the input is focused or it has a value. Otherwise it will appear on top of the input.
+   * `'stacked'`: The label will appear smaller and above the input regardless even when the input is blurred or has no value.
+   * `'fixed'`: The label has the same behavior as `'start'` except it also has a fixed width. Long text will be truncated with ellipses ("...").
```

Comment:
```suggestion
   * Where to place the label relative to the textarea.
   * `'start'`: The label will appear to the left of the textarea in LTR and to the right in RTL.
   * `'end'`: The label will appear to the right of the textarea in LTR and to the left in RTL.
   * `'floating'`: The label will appear smaller and above the textarea when the input is focused or it has a value. Otherwise it will appear on top of the input.
   * `'stacked'`: The label will appear smaller and above the textarea regardless even when the input is blurred or has no value.
   * `'fixed'`: The label has the same behavior as `'start'` except it also has a fixed width. Long text will be truncated with ellipses ("...").
```